src/screens/AddNotificationEmailScreen.tsx

```diff
@@ -83,9 +85,25 @@ class AddNotificationEmailScreen extends PureComponent<Props, State> {
     });
   };
 
+  onSuccessWhileChanging = (walletsToSubscribe: Wallet[]) => {
+    const { email } = this.state;
+    const { navigation, route, createNotificationEmail } = this.props;
+    const { onSuccess } = route.params;
```

Comment:
please insert empty lines after variables declarations. Maybe you can add tslint rule for that?

Reply:
yeah I think we should add a rule if we want that

Reply:
add some more comments like that below. Am all in for the rule

Reply:
done